README.md

```diff
@@ -13,3 +13,14 @@ This application is currently configured to use Playwright for end to end testin
 Tests should be kept in the `./tests` folder and can be run with the `npm run test` command.
 
 The `playwright.config` is used to configure tests and can be used to select which browsers we run our tests on, including mobile.
+
+## Deploying to QA environment
+This repo has a [Github Action workflow]([url](https://github.com/NYCPlanning/ae-cp-map/blob/main/.github/workflows/qa.yml)) that allows DCP engineers to deploy feature branches to a QA environment hosted at http://qa-capitalplanning.nycplanningdigital.com/. This can be helpful for getting feedback from QA and Design staff on branches _before_ they are merged into `main`. Because this environment is shared by the team, it is recommended that you confirm with other engineers that they are not using it before deploying.
```

Comment:
Two concerns with this link to the GitHub Action Workflow....

Reply:
1) GitHub doesn't seem to be interpreting it as a link in the preview: https://github.com/NYCPlanning/ae-cp-map/blob/main/.github/workflows/qa.yml

Reply:
2. Opening it locally leads to a "file not found" error. It appears to be interpreting it as a route to a local file.

On that note, I recommend making it a relative file reference- this ensures the repository is internally consistent within a version

```suggestion
This repo has a [Github Action workflow](.github/workflows/qa.yml) that allows DCP engineers to deploy feature branches to a QA environment hosted at http://qa-capitalplanning.nycplanningdigital.com/. This can be helpful for getting feedback from QA and Design staff on branches _before_ they are merged into `main`. Because this environment is shared by the team, it is recommended that you confirm with other engineers that they are not using it before deploying.
```